packages/components/src/templates/next/components/internal/Masthead/Masthead.tsx

```diff
@@ -88,60 +92,90 @@ const RestrictedHeaderBarContent = ({ children }: PropsWithChildren) => {
   )
 }
 
-const RestrictedHeaderBar = ({ children }: PropsWithChildren) => {
+const MobileMastheadButton = ({ children }: PropsWithChildren) => {
   const { isMastheadContentVisible, toggleMastheadContent } = useMasthead()
+  const buttonRef = useRef<HTMLButtonElement>(null)
+
+  const buttonProps: AriaButtonProps<"button"> = {
+    "aria-controls": MASTHEAD_CONTROL_ID,
+    "aria-describedby": "masthead-aria",
```

Comment:
nit: can we make this a `const` at the top? not super concerned cos this won't escape this file but easier for us down the line